modules/artifact-activation/src/test/java/org/mule/runtime/module/artifact/activation/internal/deployable/LightweightDeployableProjectModelBuilderTestCase.java

```diff
@@ -89,6 +92,29 @@ public void createDeployableProjectModelWithAdditionalDependenciesInAPlugin() th
     assertThat(descriptor.getArtifactId(), is("derby"));
   }
 
+  @Test
+  @Issue("W-15228973")
+  @Description("When a version is provided using pom properties, include the information when building DeployableProjectModel.")
+  public void buildDeployableProjectModelWithPomProperty() throws Exception {
+    installArtifact(getResourceFolder("apps/lightweight/application-using-additional-libraries"),
```

Comment:
add a test passing system properties that are reflected in the pom.properties

Reply:
added `DeployableProjectModelVersionParseTestCase` for the cases:
1) case that uses version from pom.properties
2) case that uses system property that was passed